assets/scss/optimized-checkout.scss

```diff
@@ -0,0 +1,180 @@
+// =============================================================================
+// Stencil Checkout - Customize the Optimized Single-Page Checkout experience
+// =============================================================================
+
+// =============================================================================
+//
+// IMPORTANT DISCLAIMER
+// Please refrain from nesting or using other class selectors than the ones given below.
+// Future support is only guaranteed if class selectors are left unchanged.
+//
+// =============================================================================
+
+//
+// Import Mixins/Tools
+// breakpoint
+// -----------------------------------------------------------------------------
+
+@import "../../node_modules/@bigcommerce/citadel/dist/tools/toolkit"; // 1
+@import "settings/global/global"; // 1
+@import "../../node_modules/@bigcommerce/citadel/dist/settings/global/global"; // 2
+
+//
+// Header Section
+// Background, Logo Position
+// -----------------------------------------------------------------------------
+
+.optimizedCheckout-header {
+    background-color: stencilColor("optimizedCheckout-header-backgroundColor");
+    // scss-lint:disable StringQuotes
+    background-image: url(stencilImage('optimizedCheckout-backgroundImage', 'optimizedCheckout-backgroundImage-size'));
+    // scss-lint:enable StringQuotes
+    background-size: cover;
+
```

Comment:
What background css properties should we use for this background image since we currently also have the ability to change the background image's size? Or should we remove the ability to change the size of the background image?
@bc-miko-ademagic @bc-annavu @icatalina @davidchin